packages/core/ui-settings/core-ui-settings-server-internal/src/types.ts

```diff
@@ -31,11 +32,12 @@ export type InternalUiSettingsServiceStart = UiSettingsServiceStart;
 
 /** @internal */
 export interface UiSettingsServiceOptions {
-  type: 'config' | 'config-global';
+  type: 'config' | 'config-global' | 'config-user';
```

Comment:
Why is this needed? My understanding was that the user settings are not actually persisted to a SO?

Reply:
While the `UiSettingsUserClient` does not currently provide functionality to save settings in SO (those methods are stubbed out), it acts like the `UiSettingsClient` and `UiSettingsGlobalClient` so if the future it could be fixed for SO settings/migrations